naming.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
```

Comment:
shouldnt this go into osbuilder-tools instead of here?

Reply:
We call it in numerous places in the Earthfile and it's readily available as part of this repo. Do we have access to osbuilder-tools everywhere in all targets?

That said, we can move this script at any point.

Reply:
just wondering, as we are removing all this kind of stuff from kairos and osbuilder-tools is the one that provides a lot of stuff for this, I thought it would make sense, but indeed we can move it afterwards.